### PR TITLE
Fixing layout of cifar10 tensors

### LINKED ISSUE #4: What order is RGBimage.into_vec() done in?
E.g. in examples/training, we assume the image.into_vec() will result in shape (3, 32, 32). It might actually be (32, 32, 3).

### PR DESCRIPTION
**Pull request by chelsea0x3b**
Resolves #4

### CODE REVIEW

**Comment by quietlychris** [comment]
@coreylowman I'm curious, are you seeing successful training on your machine? 

I've pulled these changes, and even after something like 7 training epochs am still sitting around 0.3 for accuracy, compared to the `torch_train.py` file, which gets me above 0.7.

**Comment by coreylowman** [comment]
@quietlychris yes - here's the output of the script on this branch for me:

```
Top 1 Accuracy: 0.3476
Top 1 Accuracy: 0.4797
Top 1 Accuracy: 0.5216
Top 1 Accuracy: 0.619
Top 1 Accuracy: 0.5701
Top 1 Accuracy: 0.6046
Top 1 Accuracy: 0.6359
Top 1 Accuracy: 0.6702
Top 1 Accuracy: 0.7025
```

Accuracy still not as a high as pytorch, but it's definitely getting above 0.3 for me. Are you using exactly this branch? Maybe there is a different in computing accuracy.
